spec/System/HapistranoSpec.hs

```diff
@@ -36,6 +39,48 @@ spec = do
         , "cabal install --only-dependencies -j"
         , "cabal build -j" ]
 
+  describe "fromMaybeReleaseFormat" $ do
+    context "when the first value is present" $ do
```

Comment:
Could you change the message on the test with the proper names, like `when the command line value is present`, please?.

This applies for all the new tests